src/styles/base.css

```
@@ -23,18 +25,38 @@ html {
   --sl-text-2xl: 12px;
   --sl-text-3xl: 12px;
   --sl-text-4xl: 12px;
-  --sl-text-5xl: 12px;
-  --sl-text-6xl: 12px;
+  --sl-text-5xl: 16px;
+  --sl-text-6xl: 14px;
 }
 
-h1,
 h2,
 h3,
 h4,
 h5,
 h6 {
-  font-size: 12px !important;
   text-transform: uppercase;
+  padding: 4px 8px 3px 8px;
+  display: inline-block;
+}
+
+:root[data-theme="dark"] h2,
+:root[data-theme="dark"] h3,
+:root[data-theme="dark"] h4,
+:root[data-theme="dark"] h5,
+:root[data-theme="dark"] h6 {
+  background-color: var(--sl-color-gray-5);
+}
+
+:root[data-theme="light"] h2,
+:root[data-theme="light"] h3,
+:root[data-theme="light"] h4,
+:root[data-theme="light"] h5,
+:root[data-theme="light"] h6 {
+  background-color: var(--sl-color-gray-6);
+}
+
+.sl-markdown-content p {
+  padding-left: 2ch;
```

```suggestion
.sl-markdown-content p {
  font-size: 14px;
}

.sl-markdown-content h1 {
  margin-top: 0;
}
```
Going to leave this suggestion here if you all like the larger p font without the left padding and the margin-top of h1 removed

---

I see! Yeah, makes sense. 13px looks good to me 👍